### PR TITLE
make more of the shell pass lints

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref #72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/kube-up.sh [hunk 1]
```diff
@@ -84,6 +84,7 @@ elif [[ "${validate_result}" == "2" ]]; then
 fi
 
 if [[ "${ENABLE_PROXY:-}" == "true" ]]; then
+  # shellcheck disable=SC1091
```
Comment (xichengliudui) [inline cluster/kube-up.sh]: shellcheck is used the right way
https://github.com/koalaman/shellcheck/wiki/SC1091

**Comment by xichengliudui** [comment]
@BenTheElder , How to view the details of the conflict?

**Comment by BenTheElder** [comment]
@xichengliudui github only surfaces which files are conflicting at the bottom. locally `git fetch upstream && git rebase upstream/master` should let you fix it.
`hack/.shellcheck_failures` appears to be the conflict :thinking:

**Comment by xichengliudui** [comment]
I will try. Thank you for your reply

**Pull request closed without merging**